hack/csi_test/cmd/main.go

```diff
@@ -9,8 +9,8 @@ import (
 
 var (
```

Comment:
Would it make sense to also have a help arg with more detailed examples of commands? Since it's not obvious from looking at the flags what's required for publish and what's required for unpublish

Reply:
Added a help message (it already supported -h flag), will this work?